pages/linux/schroot.md

```diff
@@ -0,0 +1,25 @@
+# schroot
+
+> Run command or interactive shell with special root directory. More customizable than standard chroot.
+> Source code: https://gitlab.com/codelibre/schroot.
+> More information: <https://wiki.debian.org/Schroot>.
+
+- Run command in a specific chroot:
+
+`schroot --chroot {{chroot}} {{command}}`
```

Comment:
`{{chroot}}` is a path? Like `{{path/to/directory}}`?

Reply:
No, it's the name of the configured schroot. A config file located at /etc/schroot/chroot.d/{name} defines the properties of the schroot. [This](https://gist.github.com/conoro/89fd8a3b538707be49221739d695db49) is what I am using a schroot for, you can look at it to see an example config file and usage.